dockerfiles/hack/run-integration.py

```diff
@@ -55,8 +56,13 @@ def build_args():
         args.extend(INTEGRATION_EXTRA_ARGS.split())
     return args
 
+def refresh_vault():
+    while True:
+        VaultClient.refresh_client()
+        time.sleep(60*10)
 
 if __name__ == "__main__":
+    threading.Thread(target=refresh_vault, daemon=True).start()
```

Comment:
we should do that only if we are using vault: https://gitlab.cee.redhat.com/service/app-interface/-/blob/ae2715fdf47097afa020842697d02d2b7af72a9c/data/app-interface/app-interface-settings.yml#L11